server/routes/bookedtoday/arrivals/singleRecordFoundController.test.ts

```diff
@@ -13,26 +14,30 @@ const expectedArrivalsService = new ExpectedArrivalsService(null, null) as jest.
 let app: Express
 
 beforeEach(() => {
-  signedCookiesProvider.mockReturnValue({
-    'new-arrival': {
-      firstName: 'Jim',
-      lastName: 'Smith',
-      dateOfBirth: '1973-01-08',
-      prisonNumber: 'A1234AB',
-      pncNumber: '01/98644M',
-      potentialMatches: JSON.stringify([
-        {
-          firstName: 'Jim',
-          lastName: 'Smith',
-          dateOfBirth: '1973-01-08',
-          prisonNumber: 'A1234AB',
-          pncNumber: '01/98644M',
-        },
-      ]),
-    },
-  })
   app = appWithAllRoutes({ services: { expectedArrivalsService }, roles: [Role.PRISON_RECEPTION] })
-  expectedArrivalsService.getArrival.mockResolvedValue(null)
+  expectedArrivalsService.getArrival.mockResolvedValue({
+    id: '1111-2222-3333-4444',
+    firstName: 'James',
+    lastName: 'Smyth',
+    dateOfBirth: '1973-01-08',
+    prisonNumber: undefined,
+    pncNumber: '01/98644M',
+    date: '2021-09-01',
+    fromLocation: 'Reading',
+    moveType: 'PRISON_REMAND',
+    fromLocationType: LocationType.COURT,
+    gender: GenderKeys.MALE,
+    isCurrentPrisoner: false,
+    potentialMatches: [
+      {
+        firstName: 'Jim',
+        lastName: 'Smith',
+        dateOfBirth: '1973-01-08',
+        prisonNumber: 'A1234AB',
+        pncNumber: '01/98644M',
+      },
+    ],
+  } as unknown as Arrival)
```

Comment:
think the unknown thing is because missing sex from potential match. It's a required field.  Shouldn't affect your test though